docs/source/explore/get-started/install.rst

```diff
@@ -53,70 +75,80 @@ To install on Windows, simply double-click on the downloaded installer file to o
-3. Drag Vircadia to the Applications folder.                                               
-4. Open **System Preferences > General**.                                                        
-5. Next to the warning indicating that Interface is blocked, click 'Open Anyway'.                
-6. Confirm that you want to open the application.                                                 
+2. Agree to the License Agreement.
+3. Drag Vircadia to the Applications folder.
+4. Try open the Interface.app. You will be prompted that macOS will not open the file because it is unsigned.
+5. Open **System Preferences > Security & Privacy**. On older macOS versions this might be in **System Preferences > General**.
+6. Next to the warning indicating that Interface is blocked, click 'Open Anyway'.
+7. Confirm that you want to open the application.
+8. Allow microphone access if prompted to be able to talk inside the application.
 
 At this point, Interface will open and you will be able to log in and begin exploring the metaverse.
 
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Linux Install
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Mark the downloaded AppImage file as executable. (for example with ``chmod +x Vircadia-Alpha-2020.2.5-Asteria.AppImage``)
+2. (optional) if you want the AppImage to be integrated into your system, install `AppImageLauncher <https://github.com/TheAssassin/AppImageLauncher>`_.
+3. Execute AppImage.
+
+Currently the server is not part of the Linux Interface AppImage. To host a domain, please refer to :doc:`Host a Domain from a Local or Cloud Linux Server <../../host/server-setup/linux-server>`.
+
 ----------------------------
 Upgrade Vircadia
 ----------------------------
 
+Vircadia is always changing, as we work to improve performance and add features that will enhance your experience in the metaverse. At any time, you can download the latest release from our `website <https://vircadia.com/download-vircadia/#interface>`_.
 
-.. Vircadia is always changing, as we work to improve performance and add features that will enhance your experience in the metaverse. If a new version has been released, you will be prompted to upgrade your installation the next time you run Vircadia. At any time, you can also download the latest release from our `website <https://www.highfidelity.com/download>`_.
-
-.. You cannot upgrade if Sandbox or the Console is running in the background of your computer. Be sure to quit these applications before upgrading.
-
-.. For Windows, locate the Vircadia app in your system tray. Right-click on the icon and select 'Quit'. Alternatively, end the 'server-console' background process using the Task Manager.
-.. For OS X, locate the Vircadia icon in the menu bar. Click on the icon and select 'Quit'.
+You cannot upgrade Vircadia on Windows if you have a server running in the background of your computer. Be sure to quit this application before upgrading.
```

Comment:
```suggestion
You cannot upgrade Vircadia on Windows if you have Vircadia Interface or 
Server running on your computer. Be sure to quit these applications before upgrading.
```

Reply:
I took the suggestion, but also added ``Keep in mind that Vircadia Server could be running in the background.``,
since users with old installations might still have this feature, or we might want to bring it back (since it is actually quite nice to run something in background).